fitnessllm_dataplatform/stream/strava/services/etl_interface.py

```
@@ -0,0 +1,230 @@
+                        element[k] = str(v)
+            return input_dict["data"]
+        return {}
+
+    @beartype
+    def upsert_to_bigquery(
+        self,
+        stream: StravaStreams,
+        dataframes: list[DataFrame],
+        metrics: list[Metrics],
+    ):
+        timestamp = datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)